README.rst

```diff
@@ -120,6 +120,8 @@ application   --application   AWS_OKTA_APPLICATION   Okta AWS application URL
 ------------- --------------- ---------------------- ----------------------------------------
 role          --role          AWS_OKTA_ROLE          AWS Role ARN
 ------------- --------------- ---------------------- ----------------------------------------
+account       --account       AWS_ACCOUNT            AWS Account Filter
```

Comment:
Lets call this `account-alias` unless we will filter by account-number as well

Reply:
agreed. fixed